webroot/adm/oncall/generate-api-key-secret.html

```diff
@@ -338,13 +336,27 @@ <h5>Create a new client</h5>
                                                 </div>
                                                 <div class="pt-3">
                                                     Roles:
-                                                    <select id="createClient-workflow-roles" class="form-select">
-                                                        <option value="" selected>Please select a role...</option>
-                                                        <option value="GENERATOR">GENERATOR - for publishers</option>
-                                                        <option value="MAPPER">MAPPER - for advertisers, data providers
-                                                        </option>
-                                                        <option value="ID_READER">ID_READER - for DSPs</option>
-                                                    </select>
+                                                </div>
+                                                <div class="form-check">
+                                                    <input type="checkbox" id="createClient-workflow-roles-GENERATOR"
+                                                        class="form-check-input" name="createClient-workflow-roles[]"
+                                                        value="GENERATOR">
+                                                    <label for="createClient-workflow-roles-GENERATOR">GENERATOR - for
+                                                        publishers123</label>
+                                                </div>
+                                                <div class="form-check">
+                                                    <input type="checkbox" id="createClient-workflow-roles-MAPPER"
+                                                        class="form-check-input" name="createClient-workflow-roles[]"
+                                                        value="MAPPER">
+                                                    <label for="createClient-workflow-roles-GENERATOR">MAPPER - for
```

Comment:
I think we need to change GENERATOR to MAPPER here: `createClient-workflow-roles-MAPPER`